canonical-websites/build.snapcraft.io
# Update buttons components to Vanilla styles

## Done

Migrated all buttons to Vanilla styled buttons (with generated CSS module).

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Buttons should have Vanilla style

## Issue / Card

Fixes #888
